### PR TITLE
[HHS + AN] House Hold Server Deployment file

### PR DESCRIPTION
**Pull request by paguos**
- Add docker images for hhs and hh-ui
- Add docker-compose for full app
- Modify application ports to docker hosts

Missing features before merging (up for debate):
- Change the URL and ports and applications from config files to env variables
- Add mock sensor

Closes issue #115

### CODE REVIEW

**Comment by paguos** [comment]
@dakingha69 some feedback? :)

File: household-ui/src/helpers/fetch.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const fetchFromEndpoint = async endpoint => {
-  const response = await fetch(`http://127.0.0.1:3002${endpoint}`, {
+  const response = await fetch(`http://household_server:3002${endpoint}`, {
```
Comment (dakingha69) [inline household-ui/src/helpers/fetch.js]: We should put this also in config file or env variable

File: household-server-config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // URL of a running MongoDB instance
-  dbUrl: "mongodb://localhost:27017",
+  dbUrl: "mongodb://mongodb:27017",
```
Comment (dakingha69) [inline household-server-config.js]: This should also be configurable via env variable maybe?

File: household-ui/src/helpers/fetch.js (hunk 1 above)
Comment (paguos) [reply household-ui/src/helpers/fetch.js]: Yep 👍 , that is what I meant in the comment of the PR 😃

**Comment by paguos** [comment]
@dakingha69 yep that is my question. Where do we want to store this config? How can we add new accounts? ... So maybe not how. But what be a good design for deploying a new household ...

**Comment by paguos** [comment]
I was thinking, maybe a single docker linux image is the way to go 🚀

**Comment by dakingha69** [comment]
Hmm yes this would make sense

**Comment by simonhuber95** [comment]
I just tried to run this in Docker, but I get the following Error:
`ERROR: Service 'authority0' failed to build: COPY failed: stat /var/lib/docker/tmp/docker-builder388857518/parity-authority/docker/parity/config: no such file or directory`

Is there something I missed? If so it should also be in the README. 
Also in the README the it states:

> Make sure you are in the parity-authority directory

But shouldn't it be

> Make sure you are in the parity-authority/docker directory

**Comment by paguos** [comment]
@simonhuber95 this branch is not ready yet ... But for the commands, you tested nope. Nothing else is needed

**Comment by paguos** [comment]
Will open a new PR will follow with the fancy docker image implementation.

**Pull request closed without merging**